smithy-openapi/src/main/java/software/amazon/smithy/openapi/model/OperationObject.java

```diff
@@ -285,27 +282,29 @@ public Builder deprecated(boolean deprecated) {
         }
 
         public Builder security(Collection<Map<String, List<String>>> security) {
-            this.security = new LinkedHashSet<>();
-            this.security.addAll(security);
+            if (this.security == null) {
```

Comment:
Might be able to use `BuilderRef::hasValue` in the ctor, then you can just make the builder member final and skip these checks